### PR TITLE
Allow paid edition databases

### PR DESCRIPTION
**Pull request by aalwash**
Like described in #60, it isn't possible to use paid edition of the databases

### CODE REVIEW

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -100,7 +100,16 @@ private function getDatabaseNode(): ArrayNodeDefinition
...
-        $database_node->children()->enumNode('edition')->values(['GeoLite2-ASN', 'GeoLite2-City', 'GeoLite2-Country']);
+        $database_node->children()->enumNode('edition')->values([
+            'GeoLite2-ASN',
+            'GeoLite2-City',
+            'GeoLite2-Country',
+            'GeoIP2-City',
+            'GeoIP2-Country',
+            'GeoIP2-Anonymous-IP',
+            'GeoIP2-Domain',
+            'GeoIP2-ISP'
```
Comment (peter-gribanov) [inline src/DependencyInjection/Configuration.php:111]: Please [fix CS](https://travis-ci.org/github/gpslab/geoip2/jobs/725876958)

```diff
-            'GeoIP2-ISP'
+            'GeoIP2-ISP',
```

**Review by peter-gribanov — CHANGES_REQUESTED** [review]

File: src/DependencyInjection/Configuration.php (hunk 1 above)
Comment (aalwash) [reply src/DependencyInjection/Configuration.php:111]: Sure, we can add the trailing comma, but are you sure about that?
It's a 7.3+ feature and the composer.json minimum php requirements are >=7.1.0

Comment (jspeedz) [reply src/DependencyInjection/Configuration.php:111]: https://wiki.php.net/rfc/list-syntax-trailing-commas

**Comment by peter-gribanov** [comment]
Thanks for your work

**Pull request merged**